doc/fitting.md

```diff
@@ -31,7 +31,7 @@ We can use these flags as follows.
 
 Taking ZTF as an example:
 
-        light_curve_analysis --model Me2017 --svd-path ./svdmodels --outdir outdir --label injection --prior priors/Me2017.prior --tmin 0.1 --tmax 20 --dt 0.5 --error-budget 1 --nlive 512 --Ebv-max 0 --injection ./injection.json --injection-num 0 --injection-outfile outdir/lc.csv --generation-seed 42 --filters g,r,i --plot --remove-nondetections --ztf-uncertainties --ztf-sampling --ztf-ToO 180
+        light_curve_analysis --model Bu2019lm --svd-path ./svdmodels --outdir outdir --label injection --prior priors/Me2017.prior --tmin 0.1 --tmax 20 --dt 0.5 --error-budget 1 --nlive 512 --Ebv-max 0 --injection ./injection.json --injection-num 0 --injection-outfile outdir/lc.csv --generation-seed 42 --filters g,r,i --plot --remove-nondetections --ztf-uncertainties --ztf-sampling --ztf-ToO 180
```

Comment:
If you are going to change Bu2019lm in the model, also need to do it in the prior.

Reply:
I changed the prior file.

"Was this the right line? I thought one had Bu2019lm model and Me2017 prior.": No actually nmma_create_injection had Bu2019lm and light_curve_analysis had  Me2017 for ZTF sampling. So I made a PR as requested by you.

Reply:
Ok but it needs to be either Me2017 or Bu2019lm in the command line, you can't have both.

Reply:
I think there's only one. I changed Me2017 to Bu2019lm. I am not keeping both.

Reply:
light_curve_analysis --model Bu2019lm --svd-path ./svdmodels --outdir outdir --label injection --prior priors/Me2017.prior --tmin 0.1 --tmax 20 --dt 0.5 --error-budget 1 --nlive 512 --Ebv-max 0 --injection ./injection.json --injection-num 0 --injection-outfile outdir/lc.csv --generation-seed 42 --filters g,r,i --plot --remove-nondetections --ztf-uncertainties --ztf-sampling --ztf-ToO 180

Has both Bu2019lm and Me2017 in the same line?

Reply:
Please check my latest commit. I have already changed the prior file.